content/webapp/components/Body/Body.tsx

```diff
@@ -287,7 +291,16 @@ const Body: FunctionComponent<Props> = ({
   };
 
   return (
-    <div className="basic-body">
+    <div
```

Comment:
Might be worth a Styled Component?

Reply:
A repo-wide search shows this is the only mention of `basic-body`. I think we can do away with it altogether. Or did you mean to use a styled-component  for the `style` object instead? I wasn't sure based on our discussion last week

Reply:
I thought it maybe be worth it since it's a more complex one (I thought we could access props.theme as well and remove that import) + classname, but you're right `basic-body` doesn't seem to be adding any styles... Not fussed about it either way!